package.json

```diff
@@ -35,7 +35,7 @@
     "babel-preset-react": "^6.5.0",
     "mocha": "^3.4.2",
     "npm-check": "^5.4.4",
-    "prop-types": "^15.5.10",
+    "prop-types": "^15.0.0",
```

Comment:
Should we remove this, because we have `prop-types` as a direct dependency already in this package?

Reply:
If you leave it as a direct dep, and not a peer dep, then for sure it should be :)